pipelines/op-release-notes.adoc

```diff
@@ -28,8 +28,10 @@ For questions and feedback, you can send an email to the product team at pipelin
 
 // Modules included, most to least recent
 include::modules/op-release-notes-1-0.adoc[leveloffset=+1]
+
```

Comment:
Is this related?

Reply:
I was getting build errors because immediately after this someone opened a multi-line comment and didn't close it with four slashes. I thought separating that chuck of text from the includes comment might help avoid that error getting re-introduced.